examples/static-hbase-encrypted/networking.tf

```diff
@@ -0,0 +1,75 @@
+module "sg-ports" {
+  # source               = "git::https://github.com/Datatamer/terraform-aws-emr.git//modules/aws-emr-ports?ref=7.3.0"
+  source       = "../../modules/aws-emr-ports"
+  applications = local.this_application
+}
+
+module "aws-emr-sg-master" {
+  source                  = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  vpc_id                  = var.vpc_id
+  ingress_cidr_blocks     = var.ingress_cidr_blocks
+  ingress_security_groups = []
+  egress_cidr_blocks      = var.egress_cidr_blocks
+  ingress_ports           = module.sg-ports.ingress_master_ports
+  sg_name_prefix          = format("%s-%s", var.name_prefix, "-master")
+  egress_protocol         = "all"
+  ingress_protocol        = "tcp"
+}
+
+module "aws-emr-sg-core" {
+  source                  = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  vpc_id                  = var.vpc_id
+  ingress_cidr_blocks     = var.ingress_cidr_blocks
+  ingress_security_groups = []
+  egress_cidr_blocks      = var.egress_cidr_blocks
+  ingress_ports           = module.sg-ports.ingress_core_ports
+  sg_name_prefix          = format("%s-%s", var.name_prefix, "-core")
+  egress_protocol         = "all"
+  ingress_protocol        = "tcp"
+}
+
+module "aws-emr-sg-service-access" {
+  source              = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  vpc_id              = var.vpc_id
+  ingress_cidr_blocks = var.ingress_cidr_blocks
+  egress_cidr_blocks  = var.egress_cidr_blocks
```

Comment:
Service access doesn't need egress cidr blocks.

You can only set a single service access security group, and the way the security group module works it'll create separate ingress and egress security groups. Since you can only set a single security group, we have to choose between egress and ingress, and for the purpose of service access, ingress makes more sense.

Reply:
Fixed.